overlord/snapstate/snapstate_update_test.go

```diff
@@ -10439,12 +10430,13 @@ func (s *snapmgrTestSuite) TestUpdateManyRevOptsOrder(c *C) {
 	testOrder := func(names []string) {
 		requestSnapToAction = nil
```

Comment:
the other approach to changing this test would be to set this to a map here, and not have it set to anything in the action response code itself, the result would keeping the last action for each snap instead involved instead? then there would be again one action per snap name

Reply:
Great suggestion, updated!